cmd/oci/main.go

```diff
@@ -50,5 +50,10 @@ func main() {
 	log.SetOutput(f)
 
 	log.Printf("OCI FlexVolume Driver version: %s (%s)", version, build)
-	flexvolume.ExecDriver(&driver.OCIFlexvolumeDriver{}, os.Args)
+
+	drivers := map[string]flexvolume.Driver{
+		"oci-bvs": &driver.OCIFlexvolumeDriver{},
```

Comment:
I suggest we use a registration mechanism for drivers like in the CCM's [`RegisterCloudProvider()`](https://github.com/oracle/oci-cloud-controller-manager/blob/75c5e87fc29d21c313cd750c99eb2eb1f3a6900b/vendor/k8s.io/kubernetes/pkg/cloudprovider/plugins.go#L44).